migrations/20190805102047-add-approvedAt-column-to-collective.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const colName = 'approvedAt';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const colParms = {
+      type: Sequelize.DATE,
+    };
+    await queryInterface.addColumn('Collectives', colName, colParms);
+    await queryInterface.addColumn('CollectiveHistories', colName, colParms);
+    return queryInterface.sequelize.query(`
+      UPDATE "Collectives" c
+        SET "approvedAt" = '2019-08-06'
```

Comment:
Sorry, also missed that until now. I think we should limit the update to `type=COLLECTIVE`. Also `HostCollectiveId IS NOT NULL`to be on the safe side.

Reply:
Okay.